### PR TITLE
Offline Flutter todo app

### PR DESCRIPTION
**Pull request by mongodben**
realm flutter CRUD todo app

### CODE REVIEW

**Review by cbush — APPROVED** [review]

File: flutter_todo/android/build.gradle [hunk 1]
```diff
@@ -0,0 +1,31 @@
+buildscript {
+    ext.kotlin_version = '1.3.50'
```
Comment (desistefanova) [inline flutter_todo/android/build.gradle:2]: It required the latest version of Kotlin 1.6.21 when I run it on Android on my environment.

File: flutter_todo/lib/components/todo_item.dart [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+      realm.write(() {
+        realm.delete(todo);
+      });
+    }
+
+    return Slidable(
+      key: const ValueKey(0),
+      endActionPane: ActionPane(motion: ScrollMotion(), children: [
+        SlidableAction(
+          onPressed: (BuildContext context) {
```
Comment (desistefanova) [inline flutter_todo/lib/components/todo_item.dart:26]: This option onPressed seems to not work on Android and I couldn't open ModifyTodoModal.

File: flutter_todo/README.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# Realm Flutter Todo
+
+A todo list application built with the [Realm Flutter SDK](https://www.mongodb.com/docs/realm/sdk/flutter/).
+
+This example only supports Flutter for Android, iOS, Windows, MacOS, and Linux.
+It **does not** support Flutter for Web.
+
+## Getting Started
+
+1. Clone and enter the project
```
Comment (blagoev) [inline flutter_todo/README.md:10]: Should we provide the commands for this step. Perhaps it's not clear what enter the project means

File: flutter_todo/pubspec.yaml [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+# To automatically upgrade your package dependencies to the latest versions
+# consider running `flutter pub upgrade --major-versions`. Alternatively,
+# dependencies can be manually updated by changing the version numbers below to
+# the latest version available on pub.dev. To see which dependencies have newer
+# versions available, run `flutter pub outdated`.
+dependencies:
+  flutter:
+    sdk: flutter
+  provider: ^6.0.0
+  uuid: 3.0.5
```
Comment (blagoev) [inline flutter_todo/pubspec.yaml:33]: We are exposing uuid through the realm package. I suggest we use that one instead

**Review by blagoev — APPROVED** [review]
looks good. I added some notes

**Review by nielsenko — APPROVED** [review]

File: flutter_todo/lib/components/create_todo.dart [hunk 5]
```diff
@@ -0,0 +1,118 @@
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+import 'package:flutter_todo/realm/schemas.dart';
+import 'package:realm/realm.dart';
+import 'package:uuid/uuid.dart';
```
[the remark this answers is not in the document]
Comment (mongodben) [reply flutter_todo/lib/components/create_todo.dart:5]: sounds good. we can always push updates to this as releases of flutter sdk come out. 

we'll also be adding flexible sync to the project once that comes out. (probably post MDBW)

**Pull request merged**